### ISSUE TITLE
What about AppCache?

### ISSUE THREAD

**Issue by peterbe** [jeffposnick/create-react-pwa#16]
ServiceWorkers are great but most of my users are iPhone safari users and/or iPhone with "Add to Home screen". 
[iOS still doesn't support Service Workers](https://webkit.org/status/#specification-service-workers) so you have to use Application Cache. Not very progressive but definitely something we have to contend with.

**Comment by jeffposnick** [comment]
I understand the iOS use case, but App Cache support is not something that will be addressed in this particular project.

**Comment by peterbe** [comment]
Understandable. Keep it simple. 

Right now I'm actually stuck on this very issue. My family and almost all my friends have iPhones and the webapp I've built is ideal for smartphones. I already have service workers figured out but almost nobody can benefit from that. 

I tried to bundle my webapp with cordova but failed on horrible xcode compilation errors. Tried phonegap (which basically wraps cordova) but it failed with the same errors too. 

I know projects like Lighthouse will actually penalize you if you have AppCache. 
Tricky situation.
